lib/rubocop-sketchup.rb

```diff
@@ -26,8 +26,8 @@
 # formatters. Naughty! Naughty!
 class RuboCop::Formatter::FormatterSet
   formatters = BUILTIN_FORMATTERS_FOR_KEYS.dup
-  formatters['extension_review'] =
-      RuboCop::Formatter::ExtensionReviewFormatter
+  formatters['extension_review'] = 'ExtensionReviewFormatter'
```

Comment:
It will then look for 'ExtensionReviewFormatter' in the `RuboCop::Formatter` namespace?

Reply:
Yes, `RuboCop::Formatter.const_get(formatter_name)` at https://github.com/rubocop/rubocop/blob/master/lib/rubocop/formatter/formatter_set.rb#L102 is calling `const_get` on the `RuboCop::Formatter` module.  The arguments to `const_get` are a symbol or a string, https://www.rubydoc.info/stdlib/core/Module:const_get. Maybe this used to accept a `Class` in Ruby 2.x but it seems to not accept that in Ruby 3.x.

Reply:
Ok. Thank you for the PR. I guess the CI tests never tests the custom formatter.

I had a quick look at RuboCop docs, and it appear to still be lacking a clear good API to add custom extensions and formatters without monkey patching some parts of RuboCop. So this has always been something that has a higher risk of breaking.